### PR TITLE
collect multiple namespaces

### PR DESCRIPTION
**Pull request by till**
- Update: support multiple namespaces
- Chore: copy right + envrc

### CODE REVIEW

**Comment by till** [comment]
Hi,

I'll add some context:

I wanted to start pulling multiple namespaces instead of deploying this importer multiple times. So here is that. By default, it will continue to work as is (hopefully), but if you leave `-namespace` empty, it will start pulling namespaces from k8s and then expose metrics for all ports.

I also added a flag `-ignore-namespaces` to _ignore_ namespaces that should not be pulled. Been running the code for a bit with a test cluster and it seems to work. Adding namespaces (and their pods) that are created and also removing namespaces (and their pods) from `/metrics`.

Thanks for sharing your project, it got me started in no time.

File: LICENSE.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Geoff Bourne
+Copyright (c) 2019-2022 Geoff Bourne
```
Comment (itzg) [inline LICENSE.txt:3]: Thanks 😀

**Review by itzg — APPROVED** [review]
This is a great addition. Thanks!

**Comment by itzg** [comment]
Released as https://github.com/itzg/kube-metrics-exporter/releases/tag/v1.2.0

**Pull request merged**